build/Dockerfile

```
@@ -61,8 +68,8 @@ RUN cd /opt/ && curl -L https://github.com/facebook/rocksdb/archive/v6.10.1.tar.
 ARG TIMEZONEINFO=America/Los_Angeles
 RUN rm -f /etc/localtime && ln -s /usr/share/zoneinfo/${TIMEZONEINFO} /etc/localtime
 
-LABEL version=0.1.18
-ENV DOCKER_IMAGEVER=0.1.18
+LABEL version=0.1.19
+ENV DOCKER_IMAGEVER=0.1.19
```

Another PR has updated the Docker version. You will need to merge in the new changes and increment the version again.
```suggestion
LABEL version=0.1.20
ENV DOCKER_IMAGEVER=0.1.20
```